controllers/project.controller.js

```diff
@@ -172,6 +159,39 @@ export async function projectDashboard(req, res) {
   }
 }
 
+export async function UpdateCollaborator(req, res) {
+     try {
+      let project=await Project.findById(req.params.projectId).populate('owner', 'name email')
+      let is_owner = String(project.owner._id) === String(req.user._id);
+      if(!is_owner){
+        response_401('The user is not the owner of project.');
+      }else{
+        let collaborators=req.body.collaborators;
```

Comment:
Do the following things too:
- Fetch existing collaborators of the project from the `Collaborators` model.
- If new collaborators are added then invite them only, and also insert them into the database.
- If any collaborator is removed, delete it from the database.